shopfloor_mobile/static/wms/src/main.js

```diff
@@ -52,52 +38,21 @@ const vuetify_themes = {
     }
 }
 
-var app = new Vue({
-    el: '#app',
+const app = new Vue({
+    router: router,
     vuetify: new Vuetify({
         // FIXME: has no effect
         // theme: {
         //     themes: vuetify_themes
         // }
     }),
     data: {
-        currentRoute: window.location.hash.slice(1),
         demo_mode: false,
         config: AppConfig,
     },
-    computed: {
-        ViewComponent () {
-            if (this.config.authenticated) {
-                // TMP hack to be able to pass around params via querystring.
-                // We'll use proper routing later.
-                return Routes.get(this.currentRoute.split('?')[0]);
-            } else {
-                return LoginPage;
-            }
-        }
-    },
     created: function () {
         this.demo_mode = window.location.hash.includes('demo')
```

Comment:
is this by error?

Reply:
The demo_mode = true
Yes because I only worked on demo.
But it may well be handled differently ?

Reply:
you have to find another way then... we must be able to switch the mode w/out changing here.
Before this change it worked w/ a `demo/` in the hash